setup.py

```diff
@@ -214,6 +214,8 @@ def main(**extra_args):
           data_files=[('share/doc/dipy/examples',
                        glob(pjoin('doc', 'examples','*.py')))],
           scripts      = [pjoin('bin', 'dipy_reconst_dti'),
+                          pjoin('bin', 'dipy_reconst_dti_restore'),
+                          pjoin('bin', 'dipy_mask'),
```

Comment:
This all seems pretty error prone. Is there some way to automate this, instead of having to add a line in here every time?